apis/README.md

```diff
@@ -1,7 +1,9 @@
 # Stargate APIs
 
-This is a root project for all Stargate V2 APIs.
-The project (including its submodules) is currently an independent project in this monorepo, and it's not connected to the root `io.stargate:stargate` project.
+This is the root project for Stargate APIs that run externally to the coordinator node.  
```

Comment:
This refers to what? What is the root project?

Reply:
will expand